File: app/src/main/java/cz/kutner/comicsdb/connector/helper/ClassifiedHelper.java
package cz.kutner.comicsdb.connector.helper;

public class ClassifiedHelper {
    public static int getCategoryId(String categoryName) {
        int categoryId = 0;
        switch (categoryName) {
            case "Prodám":
                categoryId = 1;
                break;
            case "Koupím":
                categoryId = 2;
                break;
            case "Vyměním":
                categoryId = 3;
                break;
            case "Ostatní":
                categoryId = 10;
                break;
        }
        return categoryId;
    }
}

File: app/src/main/java/cz/kutner/comicsdb/connector/helper/ClassifiedHelper.kt
package cz.kutner.comicsdb.connector.helper

public object ClassifiedHelper {
    public fun getCategoryId(categoryName: String): Int {
        var categoryId = 0
        when (categoryName) {
            "Prodám" -> categoryId = 1
            "Koupím" -> categoryId = 2
            "Vyměním" -> categoryId = 3
            "Ostatní" -> categoryId = 10
        }
        return categoryId
    }
}
